### ISSUE TITLE
Running without ARB/KHR robustness in place may crash

### ISSUE THREAD

**Issue by Terabyte1385** [utmapp/UTM#4905]
**Describe the issue**  
Replace this text with a clear and concise description of what the issue is.

**Configuration**  
* UTM Version: 4.1.5 (74)
* macOS Version: 13.1
* Mac Chip (Intel, M1, ...): M1

**Debug log** 
[debug.log](https://github.com/utmapp/UTM/files/10355445/debug.log) 

**Upload VM**  
[config.plist.zip](https://github.com/utmapp/UTM/files/10355426/config.plist.zip)

**Additional Attachments**
[QEMULauncher Crash Log](https://github.com/utmapp/UTM/files/10355439/QEMULauncher-2023-01-05-222253.ips.zip)
![Screenshot 2023-01-05 at 22 17 43](https://user-images.githubusercontent.com/78782971/210873323-feeb93a6-e66a-4830-bad0-03ff051afbdb.png)

**Notes**
It happens at random, but before it crashes, the VM hangs for about 15 seconds.

**Comment by osy** [comment]
Will look into it but in the meantime you should set your display card to “virtio-ramfb” to avoid this.

**Comment by Terabyte1385** [comment]
Now it started showing this under the same circumstances: at random, hangs before crashing
![Screenshot 2023-01-06 at 12 25 26 (2)](https://user-images.githubusercontent.com/78782971/210986536-59bdacb3-49e9-4a4b-b4c0-98e26af4622c.png)
[debug.log.zip](https://github.com/utmapp/UTM/files/10359374/debug.log.zip)
[QEMULauncher-2023-01-06-122518.ips.zip](https://github.com/utmapp/UTM/files/10359376/QEMULauncher-2023-01-06-122518.ips.zip)

**Comment by yasingunaydiin** [comment]
I got the same issue. 

It may not be the reason but: this started happening after my laptop was put on sleep mode and at night it probably bugged out bc in the morning when I opened it, I got 300 notifications that my external drive (which has my utm) hasn't been ejected properly.

**Comment by Terabyte1385** [comment]
Yeah, as I debugged more it only happens when I'm running the VM off of an external SSD. No "disk not ejected properly" notifications though.
